components/service/telemetry/src/main/java/org/mozilla/telemetry/Telemetry.java

```diff
@@ -190,6 +191,27 @@ public Telemetry recordSearch(@NonNull  String location, @NonNull String identif
         return this;
     }
 
+    /**
+     * Records the list of active experiments
+     *
+     * @param activeExperimentsIds list of active experiments ids
+     */
+    public Telemetry recordActiveExperiments(List<String> activeExperimentsIds) {
```

Comment:
Let's add a helper to `Fretboard` that returns the list of active experiments so that we can easily generate this list of Ids from it.